modules/config-sync/main.tf

```diff
@@ -14,30 +14,38 @@
  * limitations under the License.
  */
 
-module "configsync_operator" {
-
-  source = "../k8s-operator-crd-support"
-
-  cluster_name         = var.cluster_name
-  project_id           = var.project_id
-  location             = var.location
-  operator_path        = var.operator_path
-  enable_multi_repo    = var.enable_multi_repo
-  sync_repo            = var.sync_repo
-  sync_branch          = var.sync_branch
-  sync_revision        = var.sync_revision
-  policy_dir           = var.policy_dir
-  cluster_endpoint     = var.cluster_endpoint
-  create_ssh_key       = var.create_ssh_key
-  secret_type          = var.secret_type
-  ssh_auth_key         = var.ssh_auth_key
-  source_format        = var.source_format
-  hierarchy_controller = var.hierarchy_controller
+module "registration" {
+  source = "../fleet-membership"
+
+  cluster_name              = var.cluster_name
+  project_id                = var.project_id
+  location                  = var.location
+  enable_fleet_registration = var.enable_fleet_registration
+  membership_name           = var.cluster_membership_id
+}
+
+
+module "acm_operator" {
+  source = "../hub-acm-feature"
 
-  operator_latest_manifest_url  = "gs://config-management-release/released/latest/config-sync-operator.yaml"
-  operator_cr_template_path     = "${path.module}/templates/config-sync-config.yml.tpl"
-  operator_credential_namespace = "config-management-system"
-  operator_credential_name      = "git-creds"
+  enable_fleet_feature = var.enable_fleet_feature
 
-  rootsync_cr_template_path = "${path.module}/templates/root-sync.yml.tpl"
+  cluster_name          = var.cluster_name
+  project_id            = var.project_id
+  location              = var.location
+  cluster_membership_id = module.registration.cluster_membership_id
+
+  source_format = var.source_format
+  sync_repo     = var.sync_repo
+  sync_branch   = var.sync_branch
+  sync_revision = var.sync_revision
+  policy_dir    = var.policy_dir
+
+  create_ssh_key = var.create_ssh_key
+  secret_type    = var.secret_type
+  ssh_auth_key   = var.ssh_auth_key
+
+  enable_policy_controller = false
```

Comment:
Is there value in having this module vs just showcasing an example of how to use just config-sync with the acm module?

Reply:
Good point, I've consolidated into just the single `acm` module.